insecure/attacker.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package corruptible;
+
+import "google/protobuf/empty.proto";
+
+option go_package = "insecure";
+
+// to compile this file, run:
+// protoc insecure/attacker.proto --go_out=plugins=grpc:.
+
+// CorruptibleConduitFactory allows an attacker to register itself to the factory, and send messages on behalf of its conduits.
+service CorruptibleConduitFactory {
+  rpc RegisterAttacker(AttackerRegisterMessage) returns (google.protobuf.Empty) {}
+  rpc ProcessAttackerMessage(Message) returns (google.protobuf.Empty) {}
+}
+
+service Attacker {
+  rpc Observe(Message) returns (google.protobuf.Empty) {}
+}
+
+// Message is represents the message exchanged between the CorruptibleConduitFactory and Attacker services.
+message Message {
+  string ChannelID = 1;
+  bytes OriginID = 2;
+  uint32 Targets = 3;
+  repeated bytes TargetIDs = 4;
+  bytes Payload = 5;
+  string Type = 6;
```

Comment:
what is type?

Reply:
removed it in https://github.com/onflow/flow-go/pull/2033/commits/45719f9d5c61f8fb4b612a3dd7715928271434b0, as we no longer use type for encoding and decoding (it is embedded into the message).